packages/paste-website/src/pages/components/date-picker/index.mdx

```diff
@@ -0,0 +1,617 @@
+
+<LivePreview scope={{DatePicker, Label, HelpText, useUID, Anchor, Box, Stack}} noInline>
+  {StackOfPickers}
+</LivePreview>
+
+Avoid placing multiple form fields on the same horizontal row to help make it easier to scan a page vertically. Use the Grid component if you need to place form fields horizontally.
+
+### Labels and help text
+
+Labels should clearly describe the date value being requested. They should be concise and descriptive, not instructional. To do this:
+
+- Use help text to provide instruction if needed. For example, don't use "Enter the date you wish to receive your bill below" as label text. Instead, use "Billing date" as a label and "Your account will be automatically billed on the above date." as help text.
+- Avoid articles ("a", "the") and punctuation. For example, use "SIM registration code" rather than "The SIM registration code".
+
+To support internationalization, avoid starting a sentence with the label and using the field to finish it since sentence structures vary across languages. For example, use "Call log expiration date" or "How long should logs be stored?". Don't use "Remove logs after:".
+
+### Required field indicator
+
+Use required indicators to show users which fields they must fill out.
+
+<LivePreview scope={{DatePicker, Label, HelpText, useUID, Anchor}} noInline>
+  {RequiredDatePicker}
+</LivePreview>
+
+### Validation
+
+Validate form fields on form submission.
+
+Validating a form field when the user leaves the current field (on blur) can be helpful to check for syntax errors. However, this can be frustrating to users who tab through controls to navigate a page, and to screen reader users, who might not be able to detect that an error occurred on blur.
+
+Don't prevent form submission by disabling the submit button. An error message can give more information than a disabled button can to help users figure out which fields are invalid.
+
+Usage of the <inlineCode>min</inlineCode> and <inlineCode>max</inlineCode> props on Date Picker is encouraged if certain dates are invalid.
+
+<LivePreview scope={{DatePicker, Label, HelpText, useUID, Anchor}} noInline>
+  {MinAndMaxPicker}
+</LivePreview>
+
+### Errors
+
+Use inline error text to explain how to fix an error.
+
+Ideally, help text should have enough information to help users prevent errors. If help text exists and you need to show an error, the error text should replace and repeat the help text until the error has been resolved.
+
+Error text should:
+
+- Be actionable. Explain how to fix an error and if reasonable, why it happened so that it might also be prevented in the future.
+- Be concise and simple, maybe even more than normal. Avoid jargon. Try to keep error text to 2 sentences or fewer.
+- Use the passive voice for input errors to avoid placing blame on the user. For example, "A friendly name is required."
+- Use the active voice for system errors. For example, "Our systems are currently down. Please contact our support team."
```

Comment:
Do you think including a code example like for the previous sections would be helpful here, or just overkill?

Reply:
Hmm I'm leaning against the code example for this section because we already have the inline error example but could be convinced otherwise